lib/controllers/secrets.js

```diff
@@ -0,0 +1,34 @@
+const { Router } = require('express');
+const authenticate = require('../middleware/authenticate');
+const authorize = require('../middleware/authorize');
+const Secret = require('../models/Secret');
+
+
+module.exports = Router()
+  .get('/', authenticate, authorize, async (req, res, next) => {
```

Comment:
You don't need the authorize middleware here unless you are planning to update it to include the stretch goal...